src/fixate/core/config_util.py

```diff
@@ -305,14 +300,14 @@ def _print_config_dict(self, config_dict):
             self.poutput("SERIAL || " + com_port + " || " + str(parameters))
 
     def _test_print_error(self, name, msg):
-        self.poutput("ERROR: ", end="", color=RED)
-        self.poutput(str(name), end="", color=CYAN)
-        self.poutput(" - {}".format(msg))
+        self.poutput(Fore.RED + "ERROR: ", end="")
+        self.poutput(Fore.CYAN + str(name), end="")
+        self.poutput(Fore.WHITE + " - {}".format(msg))
```

Comment:
instead of directly concatenating the style, I think we should use the api provided
self.poutput(style("ERROR: ", fg="red"), end="") etc

That docs aren't super clear on what is expected...

Reply:
fg argument is the result of fg_lookup, which expects either a str or fg Enum (which is essentially Fore from colorama)